gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/02_add_shared_policy_group.yml

```diff
@@ -28,11 +28,21 @@ databaseChangeLog:
             tableName: ${gravitee_prefix}sharedpolicygroups
         - createIndex:
             tableName: ${gravitee_prefix}sharedpolicygroups
-            indexName: idx_${gravitee_prefix}sharedpolicygroups
+            indexName: idx_${gravitee_prefix}environment_id_name
             columns:
               - column:
-                  name: organization_id
+                  name: environment_id
                   type: nvarchar(64)
+              - column:
+                  name: name
+                  type: nvarchar(512)
```

Comment:
I have no opinion. I don't think it should be too small, because we've already fixed quite a bit of the subject. Because for mongo no limit and the client was limited to jdbc which was not the intention.